create_deployment.ps1

```diff
@@ -54,22 +54,25 @@ Lib\site-packages
 
 Write-Output "INFO: Set Python pth file"
 
-# Do a standard pip install of the requirements, not warning us that scripts will be unavailable
-.\python\python.exe -m pip install -r requirements.txt --no-warn-script-location
+# Do a standard pip install of the requirements and dev requirements, not warning us that scripts will be unavailable
+.\python\python.exe -m pip install -r requirements.txt -r requirements_dev.txt --no-warn-script-location
 
 Write-Output "INFO: Installed Python dependencies"
 
-# Remove folders/files that we don't need any more
-Remove-Item .\7zip -Recurse
-Remove-Item *.zip
-Remove-Item *.7z
-Remove-Item get-pip.py
-
-Write-Output "INFO: Finished cleanup"
 
 # Zip up whole folder into a zip-file with the current date in the filename
 $date_str = Get-Date -Format "yyyyMMdd"
 $output_filename = $date_str + "_pepys-import.zip"
-Compress-Archive -Path .\* -DestinationPath $output_filename
+.\7zip\7za.exe a .\$output_filename .\*
+
+Write-Output "INFO: Written zipped deployment file to $output_filename"
+
+# Remove folders/files that we don't need any more
+Remove-Item .\7zip -Recurse
+Remove-Item sqlite.zip
+Remove-Item python.zip
+Remove-Item 7zip.zip
+Remove-Item *.7z
+Remove-Item get-pip.py
 
-Write-Output "INFO: Written zipped deployment file to $output_filename"
```

Comment:
Aah, we now do the `remove-item` after the zip.  Does that mean we zip up elements that we weren't zipping up before?

Reply:
Yeah, sorry I missed that - I ended up changing things around so many times that I lost track. I've changed it now so that most of the cleanup happens before the final zip, except for the 7zip folder which we still need to keep available. I then exclude that 7zip folder from the zip file we create, and then delete it after we've created the zip file.